llvm/lib/Transforms/InstCombine/InstructionCombining.cpp

```diff
@@ -5538,8 +5539,15 @@ void InstCombinePass::printPipeline(
   OS << '>';
 }
 
+char InstCombinePass::ID = 0;
```

Comment:
missing `INITIALIZE_PASS_DEPENDENCY` below?

Reply:
I don't have a plan to support legacy pass manager.

Reply:
new PassManager Passes don't need ID IIRC.